cmd/sops/main.go

```diff
@@ -184,6 +184,14 @@ func main() {
 					Name:  "user",
 					Usage: "the user to run the command as",
 				},
+				cli.StringFlag{
+                                        Name:  "input-type",
+                                        Usage: "currently json, yaml, dotenv and binary are supported. If not set, sops will use the file's extension to determine the type",
+                                },
+				cli.StringFlag{
+					Name:  "output-type",
+					Usage: "currently json, yaml, dotenv and binary are supported. If not set, sops will use the binary output format",
```

Comment:
```suggestion
					Usage: "currently json, yaml, dotenv and binary are supported. If not set, sops will use the input file's extension to determine the output format",
```

What you wrote is partially incorrect. SOPS will use the input file's extension. If the extension is not one of the known types, binary will be used.

Reply:
Okay yeah, I see what you mean now. Thank you for the patch!